examples/seismic/tti/operators.py

```diff
@@ -5,31 +5,37 @@
 from devito.finite_differences import centered, first_derivative, transpose
 
 
-def second_order_stencil(model, u, v, H0, Hz):
+def second_order_stencil(m, damp, epsilon, delta, s, u, v, H0, Hz, **kwargs):
```

Comment:
Not sure why this change, m,damp,... all in model, no need for explicit args.
Also would avoid kwargs and just use `(.., forward=False)` as it is always needed.